flint.ui/src/components/Datepicker/DatepickerPoint.vue

```diff
@@ -63,9 +63,11 @@ export default {
         selectedStartDate.value = val
         console.log(selectedStartDate.value)
         if (date_diff.value < 0) {
-          notification.error({
+          useToast({
+            type: 'error',
+            title: 'Error',
```

Comment:
Hello @thisiskaransgit, Thanks for working on it! I would suggest to change the title from "Error" to "Invalid Start Date".

Reply:
Okay